Wrappers/Python/ccpi/viewer/CILViewer2D.py

```diff
@@ -96,6 +96,8 @@ def __init__(self, callback):
         self.AddObserver('LeftButtonReleaseEvent', self.OnLeftButtonReleaseEvent, priority)
         self.AddObserver('RightButtonReleaseEvent', self.OnRightButtonReleaseEvent, priority)
         self.AddObserver('MouseMoveEvent', self.OnMouseMoveEvent, priority)
+        self.AddObserver('CharEvent', self.OnCharEvent, priority)
```

Comment:
Here I added an observer for CharEvent. Whenever we press a key eg. "t", it triggers both a CharEvent and a KeyPressEvent